hardhat.config.ts

```diff
@@ -150,7 +150,21 @@ const config: HardhatUserConfig = {
     ],
   },
   networks: {
-    hardhat: isFork(),
+    hardhat: {
+      allowUnlimitedContractSize: true,
+      loggingEnabled: false,
+      live: false,
+    },
+    mainnetfork: {
```

Comment:
Is adding a network (`mainnetfork`) to run forked tests working here?
Because I tried the same thing as @coreyar mentioned in the [PR#181 comments](https://github.com/VenusProtocol/venus-protocol/pull/181#discussion_r1048930239) for Venus Protocol, and it was not working for me there.

Reply:
yes it works